src/Features/CSharp/Portable/CodeCleanup/CSharpCodeCleanupService.cs

```diff
@@ -111,6 +112,7 @@ internal class CSharpCodeCleanupService : ICodeCleanupService
 
             if (_codeFixServiceOpt != null)
             {
+                Logger.Log(FunctionId.Rename_InlineSession_Session, CodeCleanupLogMessage.Create(docOptions));
```

Comment:
create your own function id rather than reusing rename session

Reply:
actually, I think you need to be removed now.

Reply:
Done